src/main/resources/application.properties

```diff
@@ -1,3 +1,7 @@
+%dev.clowder.file=/tmp/my-cdappconfig.json
+# clowder.file=/tmp/my-cdappconfig.json
```

Comment:
Should this commented line be removed?

Reply:
Yes, no , perhaps. I though of it as placeholder to see what can be done. Effectively both of them can also just go.

Reply:
I would remove that and add a small section about ClowderConfigSource in README.adoc.